jackal0:
  x: 0.6
  y: 0.6
  yaw: 0

jackal1:
  x: 0.6
  y: 4.7
  yaw: 0

jackal2:
  x: 7.6
  y: 4.7
  yaw: 3.14

jackal3:
  x: 7.6
  y: 0.6
  yaw: 3.14